cmd/lakefs/application/auth.go

```diff
@@ -0,0 +1,73 @@
+package application
+
+import (
+	"context"
+	"fmt"
+	"github.com/go-ldap/ldap/v3"
+	"github.com/treeverse/lakefs/pkg/auth"
+	"github.com/treeverse/lakefs/pkg/auth/crypt"
+	"github.com/treeverse/lakefs/pkg/config"
+	"net"
+	"time"
```

Comment:
We organize the imports differently. A block for core libraries and others. See the `run.go` file in this PR for reference.